aten/src/ATen/core/function_schema_inl.h

```diff
@@ -35,7 +35,10 @@ inline std::ostream& operator<<(std::ostream& out, const FunctionSchema& schema)
   out << ") -> ";
 
   const auto& returns = schema.returns();
-  out << "(";
+  bool need_paren = returns.size() != 1 && !schema.is_varret();
```

Comment:
What does the `is_varret()` check do? (Can you paste a before/after?)

Reply:
I actually don't know. From the name and the string representation (...), I think it means variable number of return values. Also I don't see any occurrence of `...` in `native_functions.yaml`.  The `Return.parse` method in torchgen also does not handle `...`, so I think `FunctionSchema::is_varret` is something specific to the C++ FunctionSchema. 

Not sure what does pasting 'a before/after' means here since I don't change `FunctionSchema::is_varret` itself. But now the behavior is, if there is only a single return value and `FunctionSchema.is_varret` flag is *not* on, we skip the parenthesis around the return values. This will be consistent with torchgen function schema parser.

Reply:
I only find one example of using `...` in return values here for prim ops: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/runtime/register_prim_ops.cpp#L192  .

Reply:
oh thanks, that seems reasonable (not putting parens around `...`, and also the fact that none of our native_functions.yaml ops care about it)